internal/mock/clients/k8s/k8scluster/generate.go

```diff
@@ -0,0 +1,3 @@
+//go:generate go run github.com/golang/mock/mockgen -source=../../../../clients/k8s/k8scluster/cluster.go -destination=cluster.go -package k8scluster
```

Comment:
I think it would be useful to add a target for generating mocks into the Makefile, for easier usage.

Reply:
No, it is fine. I forgot about `go.generate`. Awesome!